docs/apache-airflow/howto/operator/external_task_sensor.rst

```diff
@@ -53,6 +53,16 @@ via ``allowed_states`` and ``failed_states`` parameters.
     :start-after: [START howto_operator_external_task_sensor]
     :end-before: [END howto_operator_external_task_sensor]
 
+ExternalTaskSensor with task_group dependency
+---------------------------------------------
+In Addition, we can also use the :class:`~airflow.sensors.external_task.ExternalTaskSensor` to make tasks on a DAG
+wait for another ``task_group`` on a different DAG for a specific ``execution_date``.
```

Comment:
I think you mean logical_date? execution_date is deprecated.
Lets change the ``task_group`` to reference the task group docs
example https://github.com/apache/airflow/blob/9b7e4a732496e60ef1c275c1ac17c7ed05b27b54/docs/apache-airflow-providers-google/operators/marketing_platform/analytics.rst#L44